source/shared/cpp/ObjectModel/Column.cpp

```diff
@@ -88,7 +88,7 @@ Json::Value Column::SerializeToJsonValue() const
 
     if (m_rtl.has_value())
     {
-        root[AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Rtl)] = m_rtl.value();
+        root[AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Rtl)] = m_rtl.value_or("");
```

Comment:
`value()` should be sufficient, as it has passed `has_value()`

Reply:
the compiler on iOS will fail to compile.